client/src/components/Search/style.css

```diff
@@ -0,0 +1,104 @@
+.search__form {
+  display: flex;
+  flex-direction: column;
+  max-width: 98%;
+  padding: 1rem;
+  margin: auto;
+}
+
+.place-name {
+  text-align: left;
+  width: 100%;
+  margin: auto;
+  margin-bottom: 1rem;
+}
+
+.place-name--label {
+  margin: 0;
+}
+
+.country-city-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.autocomplete-box {
+  text-align:left;
+  width:49%;
+}
+
+.type-filter__container {
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem 0;
+}
+
+.radio-group {
+  max-width: 70%;
+  margin-left: 0.25rem;
+}
+
+.button-label {
+  text-align: left;
+  margin: 0;
+}
+
+.form-action {
+  display: flex;
+  justify-content: space-around;
+}
+
+.country-dropdown {
+  margin-right: 1rem;
+}
+
+.radio-group .ant-radio-button-wrapper {
+  margin-right: 0.5rem !important;
+}
+
+.ant-radio-button-wrapper:hover {
+  background-color: var(--main-color);
+  opacity: 0.9;
+  color: #fff;
+}
+
+.ant-radio-button-wrapper:last-child {
+  border-radius: 0 !important;
+}
+
+.ant-radio-group > .ant-radio-button-wrapper-checked {
+  border: none !important;
+  background-color: var(--main-color) !important;
+  color: #fff !important;
+  font-weight: bold !important;
+}
+
+.search-btn {
+  background-color: var(--main-color) !important;
+  color: #fff !important;
+  font-size: 1.25rem !important;
+  border: none;
+  width: 34%;
+}
+
+.search-btn:hover {
+  background-color: var(--main-color);
+  color: #fff;
+  border: none;
+}
+
+.recommendation-btn {
+  background-color: var(--main-color) !important;
+  color: #fff !important;
+  border: none;
+  width: 65%;
+  opacity: var(--opacity);
```

Comment:
you need to rewrite opacity value as --opacity-search-button

Look to index file, you'll find this
```
:root {
  --main-color: #F65B1C;
  --placeholder-color: #5d686e;
  --opacity-search-button: .70;
}
```
:)